Source/Engine/Core/Config/LayersAndTagsSettings.cs

```diff
@@ -30,6 +30,33 @@ public static string[] GetCurrentLayers()
             return GetCurrentLayers(out int _);
         }
 
+        /// <summary>
+        /// The layers names.
+        /// </summary>
+        [EditorOrder(10), EditorDisplay("Terrain Layers", EditorDisplayAttribute.InlineStyle), Collection(CanResize = false, Display = CollectionAttribute.DisplayType.Inline)]
+        public string[] TerrainLayers = new string[8];
+
+        /// <summary>
+        /// Gets the current terrain layer names. Returns "Layer" + index for layers without a name.
+        /// </summary>
+        /// <returns>The layer names.</returns>
+        public static string[] GetCurrentTerrainLayers()
+        {
+    #if FLAX_TESTS
+            return Array.Empty<string>();
```

Comment:
Missing `System.Array` to fix compilation (or add using on top of the file).

Reply:
Added in https://github.com/FlaxEngine/FlaxEngine/pull/3021/commits/573b57ed4a512246bcfe2a47fa45511b7e0442ff